client/src/modules/locations/village/modal/createUpdate.html

```diff
@@ -0,0 +1,94 @@
+<form name="villageForm" bh-submit="ModalCtrl.submit(villageForm)" novalidate>
+  <div class="modal-header">
+    <ol class="headercrumb">
+      <li>
+        <span translate>VILLAGE.TITLE</span>
+
+        <label class="label label-warning text-uppercase" translate>
+          {{ModalCtrl.action}}
+        </label>
+      </li>
+    </ol>
+  </div>
+  <div class="modal-body">
+    <div class="row">
```

Comment:
Since there is a `.row` here, you have to add a block with `.col-`. Something like : 
```html
<div class="row">
  <div class="col-xs-12">
  ...
  </div>
</div>
```

Reply:
Also don't forget to correctly close `<div>` blocks with `</div>`.